content/applications/services/helpdesk/overview/receiving_tickets.rst

```diff
@@ -206,72 +147,129 @@ of when a :guilabel:`URL Regex` action is triggered (e.g., a page visit).
+if no operator`.
 
 .. note::
-   If a chatbot is added to a live chat channel, then 3 new smart buttons will appear on the channel
-   settings form: :guilabel:`Chatbots`, :guilabel:`Sessions`, and :guilabel:`% Happy`.
-
-   - The :guilabel:`Chatbots` smart button is where the chatbot can be programmed with a
-     :guilabel:`Script`. Each line in the :guilabel:`Script` contains a :guilabel:`Message`,
-     :guilabel:`Step Type`, :guilabel:`Answers`, and conditional :guilabel:`Only If` logic that
-     applies when certain pre-filled answers are chosen. To create more steps in the
-     :guilabel:`Script`, click :guilabel:`Add a line` and fill out the script steps form according
-     to the desired logic.
-   - :guilabel:`Sessions` is where live chat sessions are recorded in order of descending
-     :guilabel:`Session Date`, by default. Each record includes the :guilabel:`Attendees` involved
-     in the live chat session, the :guilabel:`# Messages`, as well as any :guilabel:`Rating` that
-     was received when the session ended.
-   - The :guilabel:`% Happy` smart button includes a log of ratings that were left by live chat
-     attendees, and are labeled by date, time, and the support agent who was responsible for the
-     live chat session.
-
-Add the URL for the pages this channel will be applied to in the :guilabel:`URL Regex` field. If
-this channel will only be available to users in specific countries, add them to the
+   If a chatbot is added to a live chat channel, a new :guilabel:`Chatbots` smart button appears on
+   the channel settings form. Click here to create and update the chatbot *script*. Each line in the
+   script contains a :guilabel:`Message`, :guilabel:`Step Type`, :guilabel:`Answers`, and
+   conditional *Only If* logic that applies when certain pre-filled answers are chosen. To create
+   more steps in the script, click :guilabel:`Add a line` and fill out the script steps form
+   according to the desired logic.
+
+Add the URLs for the pages where the channel should appear to in the :guilabel:`URL Regex` field. If
+this channel should only be available to users in specific countries, add them to the
 :guilabel:`Country` field. If this field is left blank, the channel will be available to all site
 visitors.
 
 .. image:: receiving_tickets/receiving-tickets-channel-rules.png
    :align: center
-   :alt: View of the kanban cards for the available Live Chat channels.
+   :alt: View of the Kanban cards for the available Live Chat channels.
 
 Use the live chat widget
 ************************
 
-The :guilabel:`Widget` tab on the live chat channel form offers an embeddable website widget, or a
-shortcode for instant customer/supplier access to a live chat window.
+The :guilabel:`Widget` tab on the live chat channel form offers a website widget that can be added
+to third party platforms. Additionally, a short code is available, that can provide instant access
+to a live chat window.
```

Comment:
What kind of thrid party platforms? helpdesk? CRM? examples will help here. Also-- is this short HTML code or java script? It looks to be javascript that needs to be exectuted